ngraph_onnx/onnx_importer/ops_bridge.py

```diff
@@ -739,3 +742,13 @@ def BatchNormalization(onnx_node, ng_inputs):  # type: (NodeWrapper, List[Ngraph
     epsilon = ng.broadcast(ng.constant(epsilon, dtype=get_dtype(x.get_element_type())),
                            x.shape, axis=1)
     return (scale * ((x - mean) * (1 / (ng.sqrt(var + epsilon)))) + bias)
+
+
+def Shape(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
+    """Return input shape."""
+    return ng.constant(ng_inputs[0].shape, dtype=np.int64)  # Required for ONNX unit tests.
```

Comment:
What is required by ONNX tests? This comment can refer to anything in this line starting from `return`.
Please me more specific.